packages/@webex/plugin-cc/src/services/core/EventBus.ts

```diff
@@ -0,0 +1,25 @@
+import {EventEmitter} from 'events';
```

Comment:
Here we will have our class which is an EventBus. This will basically register all the required events, and also the various handlers. When we emit events from the code, it will then be re-routed to wherver it is needed to go.
Note that I have included methods for cleanup and am cleaning it up in the tests as well.
We use a singleton setup here (static instance), and ideally there should be no memory leaks.

Open to any better way to do this (as otherwise it is difficult to route events from aqm-reqs to cc and websocket also was being injected everywhere).